.gitignore

```diff
@@ -1,6 +1,8 @@
+.idea/*
 node_modules
 env/*
 *.pem
 resumeToken
 !/env/test.env
 docker-compose.yaml
+package-lock.json
```

Comment:
According to the `npm` docs this file should be committed. @badavis what's your experience with these?